pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -1804,8 +1805,20 @@ in
   };
 
   orgmode = super.orgmode.overrideAttrs {
-    dependencies = with self; [ (nvim-treesitter.withPlugins (p: [ p.org ])) ];
+    buildInputs = [
+      git
+    ];
+    dependencies = with self; [
+      nvim-treesitter-parsers.org
+      # tree-sitter-grammars.tree-sitter-org-nvim
+      plenary-nvim
+    ];
     nvimRequireCheck = "orgmode";
+    postInstall = ''
+      mkdir -p tests/.deps/plugins
+      cp -r ${self.plenary-nvim} tests/.deps/plugins/plenary
```

Comment:
the test should just need plenary in `rtp` so instead of copying the files, start nvim with plenary in rtp. If plenary is just needed for the test runner but not the plugin, add it to checkInputs.
Feel free to bypass crazy makefiles required by Plenary test runner (plugins should use busted directly).

Reply:
this bloats the closure too !

Reply:
Yeah, I didn't like it either. I was just trying to get around their hardcoded check for finding a plugin folder in that location.